opensearch-api/lib/opensearch/api/version.rb

```diff
@@ -26,6 +26,6 @@
 
 module OpenSearch
   module API
-    VERSION = '2.1.0'.freeze
+    VERSION = '2.2.0'.freeze
```

Comment:
I know that historically our approach has been to bump the version along with a PR that requires it, but I've seen in some backport work I've been doing that this makes it harder to backport changes correctly. This feature in itself can be backported without any issue but then we would need to override the version in the backport PR manually. I know that there is additional work that happens when we backport to 1.x in that we would need to get rid of the changelog and such but just thought I'd leave this here.